pages/organization/[organizationId]/index.js

```diff
@@ -106,6 +105,18 @@ const Dashboard = () => {
     }
   }
 
+  async function fetchIsOrgAdmin () {
+    if (!router.query || !router.query.organizationId) return
```

Comment:
Might be nice to abstract away this repeated router logic into our own orgId hook in a future PR.